templates/lgi/ah212_reference.json

```diff
@@ -0,0 +1,153 @@
+            "org.rdk.OCIContainer",
+            "org.rdk.Proxies",
+            "org.rdk.SleepTimer",
+            "org.rdk.TTSResource",
+            "org.rdk.TTSSettings",
+            "org.rdk.Timer",
+            "org.rdk.UserPreferences",
+            "org.rdk.Warehouse"
+        ]
+    },
+    "hardware": {
+        "graphics": true
+    },
+    "storage": {
+        "persistent": {
+            "storageDir": "/tmp/data",
+            "maxSize": "100M"
+        }
+    },
+    "gpu": {
+        "extraMounts": [
+            {
+                "source": "/usr/share/X11/xkb",
+                "destination": "/usr/share/X11/xkb",
+                "type": "bind",
+                "options": [
+                    "rbind",
+                    "nosuid",
+                    "nodev",
+                    "ro"
+                ]
+            }],
+        "envvar": [
+            "LD_PRELOAD=/usr/lib/libwayland-client.so.0:/usr/lib/libwayland-egl.so.0"
+        ],
+        "devs": [
+            {
+                "type": "c",
+                "path": "/dev/mali0",
+                "major": 10,
+                "minor": 115,
+                "access": "rw",
+		"dynamic": true
+            },
+	    {
+                "type": "c",
+                "path": "/dev/ion",
+                "major": 10,
+                "minor": 127,
+                "access": "rw",
+                "dynamic": true
+            }
+        ],
+        "gfxLibs": [
+            {
+                "src": "/usr/lib/libEGL.so",
+                "dst": "/usr/lib/libEGL.so"
+            },
+            {
+                "src": "/usr/lib/libEGL.so",
+                "dst": "/usr/lib/libEGL.so.1"
+            },
+            {
+                "src": "/usr/lib/libGLESv2.so",
+                "dst": "/usr/lib/libGLESv2.so"
+            },
+            {
+                "src": "/usr/lib/libGLESv2.so",
+                "dst": "/usr/lib/libGLESv2.so.2"
+            },
+	    {
+                "src": "/usr/lib/libwayland-egl.so",
+                "dst": "/usr/lib/libwayland-egl.so"
+            },
+            {
+                "src": "/usr/lib/libwayland-egl.so",
+                "dst": "/usr/lib/libwayland-egl.so.0"
+            },
+            {
+                "src": "/usr/lib/libwayland-egl.so",
+                "dst": "/usr/lib/libwayland-egl.so.1"
+            },
+            {
+                "src": "/usr/lib/libessos.so.0",
+                "dst": "/usr/lib/libessos.so.0"
+            }
+        ]
+    },
+    "mounts": [],
+    "network": {
+        "options": [
+            "nat",
+            "open",
+            "private"
+        ]
+    },
+    "envvar": [
+        "XDG_RUNTIME_DIR=/tmp"
+    ],
+    "resourceLimits": [
+        {
+            "type": "RLIMIT_NPROC",
+            "hard": 300,
+            "soft": 300
+        },
+        {
+            "type": "RLIMIT_RTPRIO",
+            "hard": 6,
+            "soft": 6
+        }
+    ],
+    "dobby": {
```

Comment:
Is there a reason you added these? 

`libethanlog` is Sky's custom logging engine so I assume you don't need this!

Reply:
This was active on target host: **_PACKAGECONFIG_append_pn-dobby = " legacycomponents"_**
It was enabled to have the "bundle" command in Dobby for testing purposes.
To remove this new libethanlog dependency on DobbyInit, we have now disabled "legacycomponents" again for Dobby on target host.